gcsweb/cmd/gcsweb/templates.go

```diff
@@ -119,9 +119,9 @@ var tmplContentFooter = template.Must(template.New("content-footer").Parse(tmplC
 
 func htmlContentFooter(out io.Writer, path string) error {
 	args := struct {
-		Path    string
+		Path string
 	}{
-		Path:    path,
+		Path: path,
```

Comment:
unrelated diff?

Reply:
This diff was auto-generated as part of `make verify`.

Reply:
cc @chaodaiG I would expect go version to be pinned in `make verify` (also `make verify` shouldn't mutate the source repo??) and verify in presubmit => no change to untouched files unless the PR touches .go-version. I think we have an issue somewhere then ...

Reply:
afaict go version is pinned in `make verify`, but yes there seems like a few bugs, at least `make verify` should not mutate, please file an issue